crates/neon/src/types_impl/extract/mod.rs

```diff
@@ -184,26 +184,17 @@ pub struct Date(pub f64);
 pub trait FromArgs<'cx>: private::FromArgsInternal<'cx> {}
 
 // Convenience implementation for single arguments instead of needing a single element tuple
```

Comment:
Let's update this comment to explain that this serves as the base case for `from_args!`, and as an optimization avoids the unnecessary Node-API calls of the n > 0 cases.